Source/MOLCertificate.m

```diff
@@ -320,70 +320,82 @@ - (NSData *)certData {
 
 - (NSString *)commonName {
   return [self memoizedSelector:_cmd forBlock:^id{
-      CFStringRef commonName = NULL;
-      SecCertificateCopyCommonName(self.certRef, &commonName);
-      return CFBridgingRelease(commonName);
+    CFStringRef commonName = NULL;
+    SecCertificateCopyCommonName(self.certRef, &commonName);
+    return CFBridgingRelease(commonName);
   }];
 }
 
 - (NSString *)countryName {
   return [self memoizedSelector:_cmd forBlock:^id{
-      return [self x509ValueForLabel:(__bridge NSString *)kSecOIDCountryName
-                      fromDictionary:[self x509SubjectName]];
+    return [self x509ValueForLabel:(__bridge NSString *)kSecOIDCountryName
+                    fromDictionary:[self x509SubjectName]];
   }];
 }
 
 - (NSString *)orgName {
   return [self memoizedSelector:_cmd forBlock:^id{
-      return [self x509ValueForLabel:(__bridge NSString *)kSecOIDOrganizationName
-                      fromDictionary:[self x509SubjectName]];
+    return [self x509ValueForLabel:(__bridge NSString *)kSecOIDOrganizationName
+                    fromDictionary:[self x509SubjectName]];
   }];
 }
 
 - (NSString *)orgUnit {
   return [self memoizedSelector:_cmd forBlock:^id{
-      return [self x509ValueForLabel:(__bridge NSString *)kSecOIDOrganizationalUnitName
-                      fromDictionary:[self x509SubjectName]];
+    return ([self orgUnits].count > 0) ? [self orgUnits][0] : nil;
```

Comment:
The firstObject property on NSArray is intended for this:

return [[self orgUnits] firstObject];

Returns nil if the array is empty.